README.md

```diff
@@ -424,7 +424,7 @@ You can populate below template according to your requirements and use it as you
 
 # Notifications (email, Slack, etc.) can be sent out when a backup run finishes.
 # Configuration is provided as a comma-separated list of URLs as consumed
-# by `shoutrrr`: https://containrrr.dev/shoutrrr/v0.5/services/overview/
+# by `shoutrrr`: https://containrrr.dev/shoutrrr/v0.7/services/overview/
```

Comment:
It seems somehow the URL scheme for the docs changed, no idea why, but this link tells us to use a URL without a `v`
```suggestion
# by `shoutrrr`: https://containrrr.dev/shoutrrr/0.7/services/overview/
```

Reply:
I think the link wihtout `v` refers to latest release and with `v` to a specific version. To be able to return to the `v0.7` docs when shoutrrr has moved on to e.g. v0.8 you need to use the `v` version.
E.g. if I go to `https://containrrr.dev/shoutrrr/v0.6/services/overview/` the link exists, but if I go to `https://containrrr.dev/shoutrrr/0.6/services/overview/` I get a 404.
Currently if I go to `https://containrrr.dev/shoutrrr/0.7/services/overview/` I get the same page as `https://containrrr.dev/shoutrrr/v0.7/services/overview/`, but I expect that once shoutrrr updates the `https://containrrr.dev/shoutrrr/0.7/services/overview/` will no longer exist

Reply:
I'm mostly surprised using `v0.7` shows me this
![image](https://user-images.githubusercontent.com/1662740/235308385-ee73e10d-90c1-47d6-abbf-6452a1f5429f.png)

Reply:
I agree it's confusing. I'll change it to the link wihtout the `v` and hope it will keep working :)